kiwi_post_run

```diff
@@ -98,18 +91,23 @@ echo "version $VERSION"
 echo "release $RELEASE"
 echo "source $IMAGE"
 echo "metadata $METADATA"
+echo "tagfile $TAGFILE"
 
 # Generate metada JSON file for container-feeder
 cat << EOF > $METADATA
 {
   "image": {
     "name": "$CONTAINER_RAW",
-    "tags": [ "$CONTAINER_TAG", "latest" ],
+    "tags": [ "${VERSION}-${RELEASE}", "${CONTAINER_TAG}-${RELEASE}", "$CONTAINER_TAG", "latest" ],
```

Comment:
Yea, I was actually trying to avoid using CONTAINER_TAG at all! The CONTAINER_TAG is unreliable right now, as it's hardcoded in the kiwi spec, however it doesn't get updated whenever IBS pulls in a new version of the "core" package we decided to use to tag the image.

e.g. Our haproxy image has a 1.6.0 tag from memory, however, it contains haproxy 1.6.11 (again, from memory) - because haproxy has been updated in SLE. If we use our package version, we don't see this weirdness anymore.